Server/src/GameEngine/Application/Application.cpp

```diff
@@ -0,0 +1,18 @@
+/*
+** EPITECH PROJECT, 2024
+** R-Type Server
+** File description:
+** Application
+*/
+
+#include "Application.hpp"
+
+GameEngine::Application::Application()
+{
+    // TODO: init network multithreaded
+}
+
+void GameEngine::Application::run()
+{
+    // TODO: Start sceneManager
```

Comment:
The scene manager should be initialized at the creation of the app because you need to set up the default scene, but it's just a comment 👍

Reply:
ok :+1: